pype/plugins/ftrack/publish/collect_ftrack_family.py

```diff
@@ -0,0 +1,52 @@
+"""
+Requires:
+    none
+
+Provides:
+    instance     -> families ([])
+"""
+import os
+import pyblish.api
+
+from pype.lib.plugin_tools import filter_profiles
+
+
+class CollectFtrackFamily(pyblish.api.InstancePlugin):
+    """
+        Adds explicitly 'ftrack' to families to upload instance to FTrack.
+
+        Uses selection by combination of hosts/families/tasks names via
+        profiles resolution.
+
+        Triggered everywhere, checks instance against configured
+    """
+    label = "Collect Ftrack Family"
+    order = pyblish.api.CollectorOrder + 0.4999
+
+    profiles = None
```

Comment:
You should probably define default profile here to not break backwards compatibility:
```
{
    "hosts": [],
    "families": [],
    "task_names": [],
    "add_to_ftrack": true
}
```